src/routes/settings-route/settings-route.js

```diff
@@ -19,7 +19,7 @@ const SettingsRoute = ({
 
   useEffect(() => {
     getKbCredentials();
-  }, []);
+  }, [getKbCredentials]);
```

Comment:
This also acts like didMount, so we can disable rule here

Reply:
Done.